_i18n/nl.yml

```diff
@@ -5,6 +5,7 @@ langs:
 pages:
   about: Over
   events: Agenda
+  resources: Middelen
```

Comment:
`Middelen` or `Bronnen` :shrug:

Reply:
Personally I'd suggest 'Bronnen' in this case,  I've used it elsewhere as a translation of 'resources' too 👌